### PR TITLE
Add DDF support for Philips Hue Ensis black edition and Philips Hue Luster White and Color Ambiance within the light_zb3_C.json

### PR DESCRIPTION
**Pull request by pimw1**
Corrected version. This DDF adds support for the black version of the Philips Hue Ensis.

### CODE REVIEW

**Comment by BabaIsYou** [comment]
Can you please put this file into the manufacturer specific `/philips` subfolder rather than in the root `/devices` folder ?

**Comment by pimw1** [comment]
This is my first time ever to commit something on github. How do i do that?

**Comment by pimw1** [comment]
I've added a modified version of the DDF already into the manufacturer specific /philips subfolder

**Comment by pimw1** [comment]
...and the original new file is deleted. First time doing this, so learning on the go :)

File: devices/philips/light_zb3_C.json [hunk 1]
```diff
@@ -60,6 +64,8 @@
     "929002376101",
     "929003053301_01",
     "929003053301_02",
+    "929003052501_01",
```
Comment (ebaauw) [inline devices/philips/light_zb3_C.json:67]: Please keep the _Model Identifier_ values sorted.

**Review by ebaauw — CHANGES_REQUESTED** [review]
Otherwise this is looking good.

File: devices/philips/light_zb3_C.json (hunk 1 above)
Comment (pimw1) [reply devices/philips/light_zb3_C.json]: Done, adjusted to ascending sort order.

**Comment by pimw1** [comment]
I'm unable to understand why the workflow fails? It is refering to company folders not related to Philips. This PR is about a single json in the Philips folder.

**Comment by pimw1** [comment]
The new color variant of the Hue E14 Luster arrived. I've checked the attributes and based on this, added the model identifier to the DDF (addition of LCU001).
Tested on my installation, all working fine.

image
image
image
image

**Comment by ebaauw** [comment]
Haven’t seen cluster 0xFC04 before.  Does it stick when you re-read the _Simple Descriptor_?

What’s the _Image Type_ for this light (in the OTA panel)?

**Comment by pimw1** [comment]
The 0xFC04 cluster sticks when re-reading the _Simple Descriptor_. The _Image Type_ is 0x0000.
image

This cluster started to appear since firmware 1.104.2, and it shows for other Hue Lights as well since updating to firmware  1.104.2. For example, i see it for: 

- LCA006 (1100 lumen White and Color Ambiance E27 Bluetooth enabled). For this type, i do not see it for every light. Re-reading the _Simple Descriptor_ does not change anything.
- LCA008 (1600 lumen White and Color Ambiance E27 Bluetooth enabled)
- LCE002 (470 lumen White and Color Ambiance E14 Bluetooth enabled). For this type, i do not see it for every light. Re-reading the _Simple Descriptor_ does not change anything.
- 929002376101 (the new Iris Bluetooth enabled)
- 1746447P7 (The Hue Resonate shining both up and down)

The LCA001 (806lumen White and Color Ambiance E27 Bluetooth enabled) does not have this cluster. But i've only 1 of this type, so i can't see for sure.

What i can say for sure, is that this cluster appeared after the most recent firmware 1.104.2.

**Comment by pimw1** [comment]
After scrutinizing the results above, i see that the only lights **not** having the 0xFC04 cluster, are still on firmware 1.101.10. I'll update them now and report back the results.

**Comment by pimw1** [comment]
Everything is updated to 1.104.2, hereby the results:

- LCA006 (1100 lumen White and Color Ambiance E27 Bluetooth enabled). For this type, i see the 0xFC04 cluster for 5-out-6 lights. For 1 light, i don't see the cluster. Instead i see cluster 0xFC02 " Hue Outlet" (which i don't see in the other 5 lights). See the printscreens:
5 lights:
image

1 light:
image
image


- LCA008 (1600 lumen White and Color Ambiance E27 Bluetooth enabled): I have one light. For this light, the 0xFC04 cluster is visible.
- LCE002 (470 lumen White and Color Ambiance E14 Bluetooth enabled). For this type, i have lights. The 0xFC04 cluster is visible for all of them.
- 929002376101 (the new Iris Bluetooth enabled). The 0xFC04 cluster is visible.
- 1746447P7 (The Hue Resonate shining both up and down bluetooth enabled). The 0xFC04 cluster is visible.
- The LCA001 (806lumen White and Color Ambiance E27 Bluetooth enabled): i've 1 light bulb. The 0xFC04 cluster is not visible, instead the FC02 cluster is visible:
image

**Review by ebaauw — APPROVED** [review]

**Pull request merged**